task 2/task2/lib/main.dart

```diff
@@ -0,0 +1,196 @@
+                child: Text('Hello',
+                    style: GoogleFonts.bitter(
+                      textStyle:
+                          TextStyle(fontSize: 80, fontWeight: FontWeight.bold),
+                    )),
+              ),
+              Container(
+                padding: EdgeInsets.fromLTRB(15, 135, 0, 0),
+                child: Text('There',
+                    style: GoogleFonts.bitter(
+                      textStyle:
+                          TextStyle(fontSize: 80, fontWeight: FontWeight.bold),
+                    )),
+              ),
+              Container(
+                padding: EdgeInsets.fromLTRB(240, 135, 0, 0),
+                child: Text(',',
+                    style: GoogleFonts.bitter(
+                      textStyle: TextStyle(
+                        fontSize: 80,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.green,
+                      ),
+                    )),
+              )
+            ],
+          ),
+        ),
+        Container(
+          padding: EdgeInsets.only(top: 35, left: 20, right: 20),
+          child: Column(
+            children: <Widget>[
+              TextField(
+                decoration: InputDecoration(
+                    labelText: 'EMAIL',
+                    labelStyle: GoogleFonts.bitter(
+                      textStyle: TextStyle(
+                          fontWeight: FontWeight.bold, color: Colors.grey),
+                    ),
+                    focusedBorder: UnderlineInputBorder(
+                        borderSide: BorderSide(color: Colors.green))),
+              ),
+              SizedBox(
+                height: 20.0,
+              ),
+              TextField(
+                decoration: InputDecoration(
+                    labelText: 'PASSWORD',
+                    labelStyle: GoogleFonts.bitter(
+                      textStyle: TextStyle(
+                          fontWeight: FontWeight.bold, color: Colors.grey),
+                    ),
+                    focusedBorder: UnderlineInputBorder(
+                        borderSide: BorderSide(color: Colors.green))),
+                obscureText: true,
+              ),
+              SizedBox(
+                height: 5.0,
+              ),
+              Container(
+                alignment: Alignment(1, 0),
+                padding: EdgeInsets.only(top: 15, left: 20),
+                child: InkWell(
+                    child: Text(
+                  "Forgot Password",
+                  style: GoogleFonts.roboto(
+                      textStyle: TextStyle(
+                    color: Colors.green,
+                    fontWeight: FontWeight.bold,
+                    decoration: TextDecoration.underline,
+                  )),
+                )),
+              ),
+              SizedBox(
+                height: 20,
+              ),
+              Container(
+                height: 40,
+                child: Material(
```

Comment:
Why are using Material? You can use MaterialButton, which has all these properties. Also remove container if you are using MaterialButton